meta-luneos/recipes-connectivity/ofono/ofono_%.bbappend

```diff
@@ -9,9 +9,13 @@ PV:halium = "1.29+git${SRCPV}"
 LICENSE = "GPL-2.0-only"
 LIC_FILES_CHKSUM = "file://COPYING;md5=eb723b61539feef013de476e68b5c50a"
 
-DEPENDS += "libglibutil dbus-glib libgrilio libmce-glib"
+DEPENDS += "dbus-glib libmce-glib"
 RDEPENDS:${PN} += "mobile-broadband-provider-info ofono-conf"
 
+#For Halium 9 devices we want to use the ofono-binder-plugin, for older devices we might want to use ofono-ril-plugin
+RDEPENDS:${PN}:append:halium = " ofono-binder-plugin"
```

Comment:
@Tofee this should fix it :) I forgot an :append: there in between, so both above RDEPENDS would not get installed.